junit-platform-suite-engine/src/main/java/org/junit/platform/suite/engine/SuiteTestDescriptor.java

```diff
@@ -75,10 +75,14 @@ SuiteTestDescriptor addDiscoveryRequestFrom(Class<?> suiteClass) {
 		return this;
 	}
 
-	SuiteTestDescriptor addDiscoveryRequestFrom(UniqueId uniqueId) {
+	SuiteTestDescriptor addDiscoveryRequestFrom(Class<?> suiteClass, UniqueId uniqueId) {
 		Preconditions.condition(launcherDiscoveryResult == null,
 			"discovery request can not be modified after discovery");
-		discoveryRequestBuilder.selectors(DiscoverySelectors.selectUniqueId(uniqueId));
+		// @formatter:off
+		discoveryRequestBuilder
+				.suiteConfigurationParameters(suiteClass)
```

Comment:
JUnit Vintage probably needs an equivalent fix.

Reply:
Good point.

However, since the `JUnitPlatform` is officially deprecated in favor of the `@Suite` support, I don't think we'll make any modifications to the `JUnitPlatform` runner.